Beyond07-SOW.md

```diff
@@ -147,15 +147,15 @@ The student team mentors will set up a personal 10-minute meeting with each of
 the students in their team and give them feedback about their performance in the
 course so far: What did they do well and where should they improve. As a student
 you should take the time to think about how you perceive your own performance,
-how is that different from how the mentors see it and why. Take the opportunity
+how is different from how the mentors see it and why. Take the opportunity
```

Comment:
As far as my English goes, i think the text was better before this change.

Reply:
@brochwerger I trust your English :)
Since this PR already merged, could you please create a new PR with the changes you suggested?